htdocs/install/mysql/tables/llx_product_lot.sql

```diff
@@ -21,11 +21,11 @@ CREATE TABLE llx_product_lot (
   rowid integer AUTO_INCREMENT PRIMARY KEY,
   tms timestamp,
   fk_product integer NOT NULL,
-  batch varchar(30) NOT NULL,
-  eatby datetime DEFAULT NULL,
-  sellby datetime DEFAULT NULL,
+  fk_furnisher integer NOT NULL default 0,
```

Comment:
If goal is to provide here a foreign key to another table (i image llx_societe), field must be "NULL" with no default.